package.json

```diff
@@ -21,110 +20,76 @@
     "url": "https://github.com/AssemblyScript/assemblyscript/issues"
   },
   "dependencies": {
-    "binaryen": "102.0.0-nightly.20211028",
-    "long": "^5.2.0",
-    "source-map-support": "^0.5.20",
-    "ts-node": "^10.4.0"
+    "binaryen": "103.0.0-nightly.20211203",
+    "long": "^5.2.0"
   },
   "devDependencies": {
     "@types/node": "^16.11.6",
     "@typescript-eslint/eslint-plugin": "^5.4.0",
     "@typescript-eslint/parser": "^5.4.0",
     "diff": "^5.0.0",
+    "esbuild": "^0.14.0",
```

Comment:
Could you update to v0.14.1?

Reply:
I believe that's already in package-lock, and new installs get it anyway. Is there a command to sync package.json with it somehow so one doesn't need to do this manually each time?